Sources/Sentry/SentryProfiler.mm

```diff
@@ -64,6 +64,99 @@
     return [symbolNSStr substringWithRange:[match rangeAtIndex:1]];
 }
 
+void
```

Comment:
This diff is hard to parse since it was extracted out into a new function, what are the changes?